Update docker-compose.yml

Unicorn9x opened the pull request:
add TYPESENSE_API_KEY to immich-server / environment to be able to boot up the service or else users get an error  Error: Config validation error: "TYPESENSE_API_KEY" is required
https://github.com/immich-app/immich/issues/2034

Unicorn9x committed:
Update docker-compose.yml

add TYPESENSE_API_KEY to immich-server / environment to be able to boot up the service or else users get an error 
Error: Config validation error: "TYPESENSE_API_KEY" is required

bo0tzz commented:
This var is already in the default .env file.

Unicorn9x commented:
> This var is already in the default .env file.

it's in the wrong position tho.
I tried the default download from https://immich.app/docs/install/docker-compose => the immich server constantly throw error same as https://github.com/immich-app/immich/issues/2034

bo0tzz commented:
Position in the .env file shouldn't affect anything, can you clarify what you mean?

Unicorn9x commented:
> Position in the .env file shouldn't affect anything, can you clarify what you mean?

yes, lemme explain again.
FYI I'm using portainer and Debian bullseye

Deploy immich using this guide https://immich.app/docs/install/docker-compose => the "immich server" constantly throw error same as https://github.com/immich-app/immich/issues/2034

Error: Config validation error: "TYPESENSE_API_KEY" is required

But I already added the "TYPESENSE_API_KEY" in the .env as screenshot below:

![image](https://user-images.githubusercontent.com/19651415/227716234-1cdc2b24-fde7-49cf-8372-0462cca57b2f.png)

I figured the immich-server doesn't picked the var from .env => So I added the TYPESENSE_API_KEY=${TYPESENSE_API_KEY} to immich-server / environment then the service start to run normally without throwing error

bo0tzz commented:
As per the [portainer-specific install guide](https://immich.app/docs/install/portainer), you need to rename .env to stack.env in the compose file. That's why it isn't working for you.

Unicorn9x commented:
ohhh my bad! sorry for wasting your time... have a great weekend bro!
